postgres/datadog_checks/postgres/statements.py

```diff
@@ -181,6 +192,40 @@ def _get_pg_stat_statements_columns(self):
                 self._stat_column_cache = col_names
                 return col_names
 
+    def _check_called_queries(self):
+        pgss_view_without_query_text = self._config.pg_stat_statements_view
+        if pgss_view_without_query_text == "pg_stat_statements":
+            pgss_view_without_query_text = "pg_stat_statements(false)"
+
+        with self._check._get_main_db() as conn:
+            with conn.cursor() as cursor:
+                called_queryids = []
+                query = QUERY_COUNTS_QUERY.format(pg_stat_statements_view=pgss_view_without_query_text)
+                rows = self._execute_query(cursor, query, params=(self._config.dbname,))
+                for row in rows:
+                    queryid = row[0]
+                    if queryid is None:
+                        continue
+
+                    calls = row[1]
+                    if queryid in self._query_count_cache:
+                        diff = calls - self._query_count_cache[queryid]
+                        if diff > 0:
+                            called_queryids.append(queryid)
+                    else:
+                        called_queryids.append(queryid)
+
+                    self._query_count_cache[queryid] = calls
```

Comment:
I wonder if we should evict old queryids that was not called for several check runs. Otherwise the _query_count_cache will keep growing

Reply:
Why don't we implement it as a TTL cache?

Reply:
Yea good call.

Reply:
I added a `TTLCache` configured for 8 hours. Is this ok or do you think the TTL value and size should be conigurable?

Reply:
We usually make this value configurable, but I've never seen someone adjusting the limit. I'm agnostic about it.